api/util/responses.go

```diff
@@ -7,9 +7,10 @@ import (
 
 // Response is the JSON API response object wrapper
 type Response struct {
-	Data       interface{} `json:"data,omitempty"`
-	Meta       *Meta       `json:"meta,omitempty"`
-	Pagination *Page       `json:"pagination,omitempty"`
+	Data       interface{}  `json:"data,omitempty"`
+	Meta       *Meta        `json:"meta,omitempty"`
+	NextPage   *NextPageReq `json:"nextpage,omitempty"`
+	Pagination *Page        `json:"pagination,omitempty"`
```

Comment:
hmm... we either need to remove `Pagination` or move `NextPage` into the `Page` struct

Reply:
I think removing Pagination is the better choice. Nothing is relying on that aspect of the http api?